.bandit.yml

```diff
@@ -10,4 +10,4 @@ tests:
 # - B102
 
 skips:
-# - B101 # skip "assert used" check since assertions are required in pytests
+  - B101 # skip "assert used" check since assertions are required in pytests
```

Comment:
I was also perturbed by this.  [In another project](https://github.com/cisagov/skeleton-ansible-role/blob/develop/.pre-commit-config.yaml#L56) I opted to make bandit ignore the test directory.

We probably don't want asserts in production code, so I'm very uneasy about disabling the assert check altogether.

Reply:
I don't like disabling it either, and I didn't know about the excludes at the pre-commit level.  This is the solution!   Will modify.  Thanks.